src/php/install.sh

```diff
@@ -156,6 +197,33 @@ install_php() {
     mkdir -p $PHP_SRC_DIR
     cd $PHP_SRC_DIR
     wget -O php.tar.xz "$PHP_URL"
+}
+
+install_php() {
+    PHP_VERSION="$1"
+    PHP_INSTALL_DIR="${PHP_DIR}/${PHP_VERSION}"
+    if [ -d "${PHP_INSTALL_DIR}" ]; then
+        echo "(!) PHP version ${PHP_VERSION} already exists."
+        exit 1
+    fi
+
+    if ! cat /etc/group | grep -e "^php:" > /dev/null 2>&1; then
+        groupadd -r php
+    fi
+    usermod -a -G php "${USERNAME}"
```

Comment:
This logic is redundant in `install_previous_version` . Can we remove?

Reply:
made these changes..